package.json

```diff
@@ -35,6 +36,7 @@
     "typescript": "^2.1.5"
   },
   "dependencies": {
+    "asap": "https://github.com/alloy/asap.git",
```

Comment:
This hacked version of `asap` (to return the correct browser specific build of the lib) is needed until [this FuseBox limitation](https://github.com/fuse-box/fuse-box/issues/258) has been removed.